09/09/2016 07:19:50 PM Start logging for AFLOWpi for project Thermal in /mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi
09/09/2016 07:19:50 PM Log Level set at: logging.INFO
09/09/2016 07:19:50 PM Creating calculations in product mode
09/09/2016 07:19:50 PM Missing Pseudopotential File
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM object of type 'NoneType' has no len()
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/re.py", line 155, in sub
    return _compile(pattern, flags).sub(repl, string, count)
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/re.py", line 286, in _subx
    template = _compile_repl(template, pattern)
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/re.py", line 271, in _compile_repl
    p = sre_parse.parse_template(repl, pattern)
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/sre_parse.py", line 737, in parse_template
    s = Tokenizer(source)
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/sre_parse.py", line 192, in __init__
    self.__next()
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/sre_parse.py", line 194, in __next
    if self.index >= len(self.string):
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM Missing Pseudopotential File
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM object of type 'NoneType' has no len()
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/re.py", line 155, in sub
    return _compile(pattern, flags).sub(repl, string, count)
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/re.py", line 286, in _subx
    template = _compile_repl(template, pattern)
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/re.py", line 271, in _compile_repl
    p = sre_parse.parse_template(repl, pattern)
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/sre_parse.py", line 737, in parse_template
    s = Tokenizer(source)
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/sre_parse.py", line 192, in __init__
    self.__next()
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/CONDA/lib/python2.7/sre_parse.py", line 194, in __next
    if self.index >= len(self.string):
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 2299, in maketree
    os.mkdir(ddir)
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/bab643c192d2b43a_01.in'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 2321, in maketree
    f = open(finp,'w')
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1303, in _writeTemplate
    with open(finp,'w') as fileName:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 6429, in _findInBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 6429, in _findInBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 6429, in _findInBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 6429, in _findInBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 6429, in _findInBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 6429, in _findInBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1553, in _addToBlock
    with open(os.path.join(subdir,fileName),'r') as inputfile:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
09/09/2016 07:19:50 PM [Errno 2] No such file or directory: '/mnt/home/supka1arCMICH/AFLOWpi/examples/Beginner/Thermal/Thermal/Si/AFLOWpi/Thermal/Si/Thermal_Si_0001/_bab643c192d2b43a_01.py'
09/09/2016 07:19:50 PM [1;31mERROR:   File "/mnt/home/supka1arCMICH/.local/lib/python2.7/site-packages/AFLOWpi/prep/prep.py", line 1564, in _addToBlock
    with open(os.path.join(subdir,'_'+ID+'.py'),'w') as outfileObj:
[0m
09/09/2016 07:19:50 PM [1;31mERROR: %%%DEBUG%%%[0m
